esx_service/vmdk_ops.py

```diff
@@ -1636,7 +1636,12 @@ def main():
 
         kv.init()
         connectLocalSi()
+
+        # start the daemon. Do all the task to start the listener through the daemon
+        threadutils.start_new_thread(target=vm_listener.start_vm_changelistener,
+                                 daemon=True)
```

Comment:
Since we have already connected to the local SI do we still need to have the loops to connect to hostd in the listener. I mean hostd is already available at this point.